wgpu-hal/src/lib.rs

```diff
@@ -1083,6 +1083,19 @@ pub struct ComputePassDescriptor<'a> {
     pub label: Label<'a>,
 }
 
+pub enum RawInstance {
```

Comment:
if it's only needed by wgpu-core, it should be defined there, not here

Reply:
This code is already outdated.